### DISCUSSION TITLE
Tasmota slow to update mqtt with wifi 6

### DISCUSSION THREAD

**Discussion by chriswalker01** [arendst/Tasmota#14820] [Q&A] [answered]
Hi,

I have 41 Tasmota devices dotted around my house. They are all either 9.3.1 or 10.1.0.

Until recently I was using a very old Asus RT N-12 without issue.

I then "upgraded" to a Ubiquiti Lite 6 and the performance of my Tasmota devices dropped. It appears to only affect updates from the devices to MQTT. Commands sent from MQTT to the devices are still fast.

I found some posts discussing issues with wifi 6 and Tasmota, but nothing seemed to match my exact problem. Still, as a troubleshooting technique, I have now installed a Ubiquiti AC Pro and my Tasmota devices are all back to normal and working well.

I have fiddled with every setting I can find on the Lite 6 and can't get any improvment. I have the two access points (AC Pro and Lite 6) in exactly the same place (only ever one plugged in at a time) and with exactly the same wifi settings.

Another oddity is, apparently Ubiquiti only upgraded the 5GHz to wifi 6. As far as I can well, the 2.4GHz is the same on the AC Pro and the Lite 6, albeit with fewer antennas.

So, I have two theories. Is the Lite 6 just not up to the job? Is it the extra antennas that made the difference? If so, I'm surprised my 12 year old RT-N12 could out perform the Lite 6.

Or, is it a bug in the Ubiquiti firmware?

Or, is there something I've missed or a config issue?

I'll keep the AC Pro going for the moment and will probably sell the Lite 6 if I can't find the issue, but does anyone have any ideas or suggestions?

Please let me know if you need anymore info.

**Comment by chriswalker01** [comment]
Does anyone have any suggestions?
I've followed the wifi 6 related tips on https://tasmota.github.io/docs/FAQ/.

The fact that only Tasmota devices are affected and that it only happens in one direction makes me think it must be something in Tasmota or something related to the TYWE3S chips that I'm using.

**Comment by Jason2866** [comment, marked as answer]
Nothing can be done from Tasmota side, sice it uses only the underlying (closed source) wifi software stack from espressif. If you can not find a working setup for the AP hardware the only way is to switch this.
